renku/core/models/provenance/activities.py

```diff
@@ -550,7 +550,6 @@ def from_run(cls, run, client, path, commit=None, subprocess_index=None, update_
 
         process_run.generated = generated
 
-        process_run.plugin_annotations()
```

Comment:
@Panaetius tagging you because based on git blame you've added this to the src. i think this is an unnecessary extra call of the plugin as it has been handled already in the post init function. or maybe i'm misinterpreted things here...?

Reply:
This seems unnecessary to me as well since we don't store the results anywhere.